### PR TITLE
TDP-4003 update member

### PR DESCRIPTION
**Pull request by rmello-daitan**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4003-new.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4003-new.s3-website-us-east-1.amazonaws.com/

File: voice/methods/conferences/putConferencesConferenceIdMembersMemberId.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+{% sample lang="http" %}
+
+```bash
+curl -X PUT \
+    --url 'https://voice.bandwidth.com/api/v2/accounts/{accountId}/conferences/{conferenceId}/members/{memberId}' \
+    -u '{username}:{password}' \
+    -H 'Content-type: application/json' \
+    --data-raw '
+    {
+      "mute"       : "true"
```
Comment (n-miles) [inline voice/methods/conferences/putConferencesConferenceIdMembersMemberId.md]: This shouldn't have quotes right?

File: voice/methods/conferences/putConferencesConferenceIdMembersMemberId.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter       | Description                                                                  | Mandatory |
+|:----------------|:-----------------------------------------------------------------------------|:----------|
+| mute            | (optional) If true, member can't speak in the conference                     | No        |
+| hold            | (optional) If true, member can't speak or hear in the conference             | No        |
```
Comment (n-miles) [inline voice/methods/conferences/putConferencesConferenceIdMembersMemberId.md]: Please specify what happens if you omit each param

File: voice/methods/conferences/putConferencesConferenceIdMembersMemberId.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+{% method %}
+## Update active Conference Member
+Update properties of an active conference member.
+
+### Request URL
+
+<code class="post">PUT</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/conferences/{conferenceId}/members/{memberId}`
```
Comment (muesono-daitan) [inline voice/methods/conferences/putConferencesConferenceIdMembersMemberId.md]:
```suggestion
<code class="put">PUT</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/conferences/{conferenceId}/members/{memberId}`
```

File: voice/methods/conferences/putConferencesConferenceIdMembersMemberId.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+```json
+{
+    "mute" : true
+}
+```
+
+```
+HTTP/1.1 204
+```
```
Comment (muesono-daitan) [inline voice/methods/conferences/putConferencesConferenceIdMembersMemberId.md]: We are returning 204, so we shouldn't have this json.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4003-new.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4003-new.s3-website-us-east-1.amazonaws.com/

**Review by n-miles — APPROVED** [review]

**Review by muesono-daitan — APPROVED** [review]

**Pull request merged**